requirements.txt

```diff
@@ -0,0 +1,10 @@
+boto3
```

Comment:
Let's take this opportunity to fix these dependencies and make them easier to update. Would you mind making a requirements.in file with he high level (unpinned) dependencies, and running `pip-compile --output-file requirements.txt requirements.in` to create the requirements.txt file with the pinned dependencies?

More context: https://github.com/jazzband/pip-tools

Reply:
I'm having trouble running this command. Do I need to setup a python virutalenv outside of the project and use that?

Reply:
Yes, you can do that. `pip install pip-tools` should install `pip-compile` for you. 

$ virtualenv env2
New python executable in /Users/ccastrapel/localrepos/consoleme/env2/bin/python
Installing setuptools, pip, wheel...done.
$ . env2/bin/activate
$ pip install pip-tools
Collecting pip-tools
  Using cached pip_tools-1.11.0-py2.py3-none-any.whl
Collecting first (from pip-tools)
  Using cached first-2.0.1-py2.py3-none-any.whl
Requirement already satisfied: setuptools in ./env2/lib/python2.7/site-packages (from pip-tools)
Collecting six (from pip-tools)
  Using cached six-1.11.0-py2.py3-none-any.whl
Collecting click>=6 (from pip-tools)
  Using cached click-6.7-py2.py3-none-any.whl
Installing collected packages: first, six, click, pip-tools
Successfully installed click-6.7 first-2.0.1 pip-tools-1.11.0 six-1.11.0
$ pip-compile --output-file requirements.txt requirements.in